templates/partials/participant-patient-status.html.twig

```diff
@@ -4,6 +4,11 @@
             <div class="panel-heading">
                 <h3 class="panel-title">Patient Status: {{ app.session.get('siteOrganizationDisplayName') }}</h3>
             </div>
+            {% import 'macros/display-text.html.twig' as macros %}
+            {{ macros.displayFalshMessage('patient-status') }}
+            {% for flashMessage in app.session.flashbag.get('patient-status-success') %}
+                <div class="alert alert-success well-sm text-center">{{ flashMessage }}</div>
+            {% endfor %}
```

Comment:
Isn't this handled by the macro above?

Reply:
I forgot to remove this after creating the macro. Will remove that.